src/example-relay/next.config.js

```diff
@@ -3,35 +3,40 @@
 const path = require('path');
 const withTranspileModules = require('next-transpile-modules');
 const withCustomBabelConfigFile = require('next-plugin-custom-babel-config');
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+});
 
 const getTranspileWorkspaces = require('./scripts/getTranspileWorkspaces'); // @x-shipit-disable
 
 const transpileWorkspaces = getTranspileWorkspaces(); // @x-shipit-disable
 // @x-shipit-enable: const transpileWorkspaces = [];
 
-module.exports = (withCustomBabelConfigFile(
-  withTranspileModules(transpileWorkspaces)({
-    images: {
-      domains: ['images.kiwi.com'],
-    },
-    babelConfigFile: path.join(
-      __dirname,
-      // @x-shipit-enable: 'babel.config.js',
-      '.babelrc.js', // @x-shipit-disable
-    ),
-    webpack: (config) => {
-      config.module.rules.push({
-        type: 'javascript/auto',
-        test: /\.mjs$/,
-      });
+module.exports = (withBundleAnalyzer(
```

Comment:
Tip: there is a great `next-compose-plugins` that makes the Next.js configuration much more approachable, see: https://github.com/adeira/universe/blob/c96ff38e23de5eb9662ae7a3fdf160aa1c17d8c6/src/abacus-backoffice/next.config.js

Reply:
Nice, I was thinking something like that would be nice. I'll add a separate PR for it.